x-pack/plugins/security_solution/public/flyout/right/components/investigation_guide.tsx

```diff
@@ -69,8 +56,10 @@ export const InvestigationGuide: React.FC = () => {
           </h5>
         </EuiTitle>
       </EuiFlexItem>
-      <EuiFlexItem>
-        {!error && basicAlertData.ruleId && ruleNote ? (
+      {loading ? (
+        <FlyoutLoading data-test-subj={INVESTIGATION_GUIDE_LOADING_TEST_ID} />
```

Comment:
I know this wasn't the original goal of the PR (well it kinda was but not exactly), but what do you think about using a [EuiSkeleton](https://eui.elastic.co/#/display/skeleton) here instead of a loading spinner?